modules/ROOT/pages/deployment/nfs.adoc

```diff
@@ -1 +1,271 @@
+:innodb_flush_method-url: https://mariadb.com/kb/en/library/innodb-system-variables/#innodb_flush_method
+
+:description: ownCloud recommends using NFS for any scenario when accessing Unix based remote servers or storages other than local storage. It has solid performance and is very stable. This guide covers the official ownCloud NFS (Network File System) deployment recommendations for oCIS.
+
+== Introduction
+
+{description}
+
+NOTE: This guide only covers the NFS client side where ownCloud runs. Follow the server or storage vendors recommendations to configure the NFS server (storage backend).
+
+== NFS Prerequisites
+
+See the xref:prerequisites/index.adoc#nfs_notes_prerequisites[NFS Notes] in the prerequisites documentation for important details.
+
+== General Performance Considerations
+
+It is advised to use network storage like NFS only in un-routed, switched Gigabit or higher environments.
+
+Please consider that a network stack runs in ranges of µs while a storage backend usually runs in ranges of ms.
+Any tuning considerations should therefore, beside mounting options, first be attempted on the backend storage layout side, especially under high loads.
+
+== NFSv4 Overview
+
+To use NFS for oCIS, you must use *NFSv4* for various reasons. 
+
+* Most important, starting with NFSv4, *Extended Attributes* can be provided by the NFS server, _if implemented_. See the xref:prerequisites/index.adoc#nfs_notes_prerequisites[NFS Notes] in the prerequisites documentation for more details.
+* *Improved Security:* It {nfs-strong-security-architecture-url}[mandates a strong security architecture]. It does not require {rpc-statd-url}[rpc.statd] or {lockd-url}[lockd]. As a result, it only uses port 2049.
+* *Improved Reliability:* Uses TCP by default.
+* *Improved Performance:* It uses Multi-Component Messages, which reduce network traffic. It is capable of using a 32KB page size, compared to the default, 1024 bytes.
+* Use of {nfs-read-write-delegations-url}[Read/Write Delegations].
+
+[caption=]
+.NFSv4 Key Value Overview
+[width="100%",cols="30%,70%",options="header",]
+|===
+
+| NFSv4
+| Explanation
+
+| Exports
+| All exports can be mounted together in a directory tree structure as part of a pseudo-filesystem
+
+| Protocol
+| A single protocol with the addition of OPEN and CLOSE for security auditing
+
+| Locking
+| Lease based locking in the same protocol
+
+| Security
+| Kerberos and ACL based
+
+| Communication
+| Multiple operations per RPC. (Improves performance)
+
+| I18N
+| UTF-8
+
+| Scalable parallel access to files distributed among multiple servers
+| pNFS (with NFSv4.1)
+
+| Extended Attributes
+| Introduced by {ea-ietf-url}[IETF, December 2017]
+|===
+
+== NFS Mount Options
+
+See the {man-nfs-ubuntu-url}[Ubuntu 20.04 man pages] for a detailed description of the NFS mount options. The following options are default for NFS except if explicitly set differently when mounting: `rw`, `suid`, `dev`, `exec`, `auto`, `nouser`, and `async`.
```

Comment:
Is there a reason to point to the Ubuntu man pages instead of the man page project?
https://man7.org/linux/man-pages/man5/nfs.5.html

Reply:
no, link and text can be changed